### PR TITLE
Fix Github button 'suggest edit'  (at least for non-submodule docs)

### PR DESCRIPTION
**Pull request by ksuess**

### CODE REVIEW

**Review by stevepiercy — APPROVED** [review]

**Comment by ksuess** [comment]
Fixing the "suggest edit" link for submodule docs would mean a massive interference in sphinx-book-theme https://github.com/executablebooks/sphinx-book-theme/blob/526956165f3fda3e5faaf53ffc39081715702f6d/src/sphinx_book_theme/header_buttons/__init__.py#L94-L102

I tend to just hide the link for submodule pages (Volto, plone.restapi, plone.api) via CSS.

**Comment by stevepiercy** [comment]
What do you think of using JavaScript to rewrite the URL for non-main documentation to https://6.dev-docs.plone.org/contributing/index.html ?

Actually, I think I would prefer to use that link site-wide than "suggest an edit" for each page. I don't want to raise expectations of people making "drive by" pull requests without signing the Plone Contributor Agreement. This would direct them toward the authoritative method to submit a PR.

**Comment by ksuess** [comment]
> What do you think of using JavaScript to rewrite the URL for non-main documentation to https://6.dev-docs.plone.org/contributing/index.html ?
> 
> Actually, I think I would prefer to use that link site-wide than "suggest an edit" for each page. I don't want to raise expectations of people making "drive by" pull requests without signing the Plone Contributor Agreement. This would direct them toward the authoritative method to submit a PR.

Yes, I completly aggree. 👍

File: docs/conf.py [hunk 1]
```diff
@@ -214,7 +214,8 @@
     # "google_analytics_id": "UA-1907133-6",
     "path_to_docs": "docs",
     "repository_url": "https://github.com/plone/documentation",
-    "repository_branch": "main",
+    # TODO: Going live: Switch "repository_branch" to 'main' (Github buttons in header)
+    "repository_branch": "6-dev",
     "use_repository_button": True,
     "use_issues_button": True,
     "use_edit_page_button": True,
```
Comment (stevepiercy) [inline docs/conf.py:221]:
```suggestion
    "use_edit_page_button": False,
```

Then in a template, we can hard code a link to https://6.dev-docs.plone.org/contributing/index.html

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
See https://github.com/plone/documentation/issues/1211

**Comment by MrTango** [comment]
> What do you think of using JavaScript to rewrite the URL for non-main documentation to https://6.dev-docs.plone.org/contributing/index.html ?
> 
> Actually, I think I would prefer to use that link site-wide than "suggest an edit" for each page. I don't want to raise expectations of people making "drive by" pull requests without signing the Plone Contributor Agreement. This would direct them toward the authoritative method to submit a PR.

For the docs it is not required to sign the controbutor aggrement as far as i know.
With that in mind, i would prefer a direct link to update docs, not just to a document.

**Comment by stevepiercy** [comment]
> For the docs it is not required to sign the controbutor aggrement as far as i know.

See https://6.dev-docs.plone.org/contributing/index.html#granting-permission-to-publish as established by https://docs.plone.org/about/license.html.

> With that in mind, i would prefer a direct link to update docs, not just to a document.

I would, too. However with the complexity of importing submodules from three other repos, this is not easy.

**Comment by ksuess** [comment]
GitHub link for "suggest edit" overridden for this repo: Impede drive-by edits by linking chapter on how to contribute. Fixed in https://github.com/plone/documentation/pull/1292

**Pull request closed without merging**